package.json

```diff
@@ -105,7 +105,7 @@
     "@storybook/addon-viewport": "5.1.0-rc.0",
     "@storybook/react": "5.1.0-rc.0",
     "@storybook/theming": "5.1.0-rc.0",
-    "@types/enzyme": "^3.9.1",
+    "@types/enzyme": "^3.9.3",
```

Comment:
In the end, this upgrade didn't fix the enzyme bug but I'm keeping it in to keep us up to date